src/modm/board/thingplus_rp2040/module.lb

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2022, Andrey Kunitsyn
+# Copyright (c) 2022, Nikolay Semenov
+#
+# This file is part of the modm project.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+# -----------------------------------------------------------------------------
+
+def init(module):
+    module.name = ":board:thingplus-rp2040"
+    module.description = """\
+# SparkFun Thing Plus - RP2040
+
+The SparkFun Thing Plus - RP2040 is a low-cost, high performance board
+with flexible digital interfaces featuring the Raspberry Pi Foundation's RP2040 microcontroller.
+Besides the Thing Plus or Feather footprint (with 18 GPIO pins),
+the board also includes an SD card slot, 16MB (128Mbit) flash memory,
+a JST single cell battery connector (with a charging circuit and fuel gauge sensor),
+an addressable WS2812 RGB LED, JTAG PTH pins, four (4-40 screw) mounting holes,
+and SparkFun's Qwiic connector.
+
+https://www.sparkfun.com/products/17745
+
+See `modm:rp-pico` for programming instructions.
```

Comment:
```suggestion
See `modm:board:rp-pico` for programming instructions.
```

Reply:
It doesn't matter because both [`modm:board:rp-pico`](https://modm.io/reference/module/modm-board-rp-pico/) and [`modm:rp-pico`](https://modm.io/reference/config/modm-rp-pico/) share the same documentation.